java-manta-client/src/main/java/com/joyent/manta/client/crypto/EncryptingPartEntity.java

```diff
@@ -89,7 +89,7 @@ public EncryptingPartEntity(final OutputStream cipherStream,
 
     @Override
     public boolean isRepeatable() {
-        return wrapped.isRepeatable();
+        return false;
```

Comment:
The underlying Apache HTTP Client library uses this method to determine if it can automatically retry a request. `MantaHttpRequestRetryHandler` is only responsible for determining whether a request should be retried based on the exception thrown. The `HttpEntity` also gets to decide whether it is eligible for automatic retries by returning a value here. [See HttpClient source for more context](https://github.com/apache/httpcomponents-client/blob/4.5.x/httpclient/src/main/java/org/apache/http/impl/execchain/RetryExec.java#L106)

Having this be true or delegate to the wrapped instance means that the HttpClient can invoke `writeTo` on the `HttpEntity` for certain kinds of uploads and not others when in fact it should never automatically retry an upload that could affect `Cipher` state